draft-irtf-cfrg-opaque.md

```diff
@@ -2127,10 +2126,11 @@ disclose their passwords to the server, even during registration. Note that a co
 server can run an exhaustive offline dictionary attack to validate guesses for the client's
 password; this is inevitable in any aPAKE protocol. (OPAQUE enables defense against such
 offline dictionary attacks by distributing the server so that an offline attack is only
-possible if all - or a minimal number of - servers are compromised {{JKX18}}.) Furthermore,
-if the server does not sample this OPRF key with sufficiently high entropy, or if it is not
-kept hidden from an adversary, then any derivatives from the client's password may also be
-susceptible to an offline dictionary attack to recover the original password.
+possible if all - or a minimal number of - servers are compromised {{JKX18}}. The details
```

Comment:
Can we keep it still as just one sentence?

`...servers are compromised {{JKX18}}. The details...`

-> 

`...servers are compromised {{JKX18}}, but the details...`

Reply:
Sure